tests/test_cloudpickle_wrapper.py

```diff
@@ -200,3 +194,11 @@ def test_func(x):
                 check_subprocess_call(cmd, stdout_regex=r'ok', timeout=10)
         finally:
             os.unlink(filename)
+
+    def test_set_loky_pickler_failures(self):
+
+        with pytest.raises(ValueError, match=r"Failed to import 'no_module'"):
+            set_loky_pickler("no_module")
+
+        with pytest.raises(ValueError, match=r"Failed to find Pickler"):
```

Comment:
`Module "os"  has no attribute "Pickler"`

You could even raise attribute error directly.